### PR TITLE
Config ~> 1.2 supports configuring settings via environment variables

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -786,7 +786,7 @@ DEPENDENCIES
   byebug
   capybara
   coffee-rails (~> 4.1.0)
-  config
+  config (~> 1.2)
```
Comment (jcoyne) [inline Gemfile.lock]: The version of config that was installed hasn't been changed. It's good that we're being more specific with what we want though.

File: config/settings.yml [hunk 2]
```diff
@@ -1,23 +1,23 @@
 multitenancy:
   enabled: false
-  default_host: "<%= ENV.fetch('SETTINGS__MULTITENANCY__DEFAULT_HOST', '%{tenant}.dev') %>"
+  default_host: "%{tenant}.dev"
```
Comment (jcoyne) [inline config/settings.yml]: Would it be helpful to write something either as a comment in this file, or in the readme explaining how use env variables to override the settings here?

Comment (cbeer) [inline config/settings.yml]: 👌

Comment (mjgiarlo) [inline config/settings.yml]: I'd like to see something in the readme about this.

Comment (cbeer) [inline config/settings.yml]: I disagree; I'm not sure there's much value in polluting our readme with implementation details about upstream gems (although a sentence saying "there's a bunch of settings in `config/settings.yml` seems reasonable, but I'd argue unrelated to this PR...).

**Pull request merged**